src/RoverExplorer/RoverExplorer.module.scss

```diff
@@ -1,3 +1,112 @@
-.roverExplorer {
-    background-color: blue;
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani:wght@700&display=swap');
+@import "../generalStyle.scss";
+
+@mixin GridItem {
+    align-items: center;
+    text-align: center;
+    vertical-align: middle;
+    line-height: 200px;
+    font-size: 20px;
+    color: $colorTextLight;
+    border-radius: 3%;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    border-color: white;
+    border-style: inset;
+    border-width: 0.1px;
+    height: 100%;
+    @include subtitle;
+}
+
+
+
+.Content {
+    background: url("/images/rover_explorer_page_background.jpg");
+    min-height: 825px;
+    background-position: center;
+    background-size: cover;
+    position: relative;
+
+}
+
+
+.GridItem1 {
+    @include GridItem;
+    grid-area: one;
+    background: url("/images/perserverance_rover.png");
+}
+
+.GridItem2 {
+    @include GridItem;
+    grid-area: two;
+    background: url("/images/oppertunity_rover.png");
+}
+
+.GridItem3 {
+    @include GridItem;
+    grid-area: three;
+    background: url("/images/spirit_rover.png");
+}
+
+.GridItem4 {
+    @include GridItem;
+    grid-area: four;
+    background: url("/images/curiosity_rover.png");
+}
+
+.GridItem5 {
+    @include GridItem;
+    grid-area: five;
+    background: url("/images/view_all_images_cover.jpg");
+}
+
+
+
+.GridContainer {
+    display: grid;
+    grid-template-areas:
+        'one one  two two'
+        ' three three four four'
+        ' five five five five';
+    grid-gap: 20px;
+    grid-template-rows: 200px 200px 175px;
+    padding: 10%;
+    max-width: 1200px;
+    left: 0;
+    right: 0;
+    margin: auto;
+    height: 0;
+    position: relative;
+
+}
+
+.Header {
+    position: relative;
+    height: 100px;
+    background-color: #ffffff;
+    opacity: 0.65;
+    margin:auto;
+   
+   
+}
+
+.HeaderText {
+    position: absolute;
+    top: 30%; // (may cause problems later) this should be 50% to get the text in the middle of the header but for some reason this works and 50% dosen't.
```

Comment:
it there a margin / padding on the text that's causing it to be nudged downwards a little?